snippets/csharp/query_points.cs

```diff
@@ -41,7 +41,7 @@
 			Limit = 20
 		}
 	},
-	query: Fusion.Rrf
+	query: new FusionQuery { Fusion = Fusion.Rrf }
```

Comment:
```suggestion
	query: Fusion.Rrf
```
Should work because of https://github.com/qdrant/qdrant-dotnet/blob/eb90d2b5e0c661f4aaa888b1a1860ec154fd8cf5/src/Qdrant.Client/Grpc/Query.cs#L29-L34.